vendor/github.com/quobyte/api/rpc_client.go

```diff
@@ -112,7 +113,17 @@ func (client QuobyteClient) sendRequest(method string, request interface{}, resp
 	defer resp.Body.Close()
 
 	if resp.StatusCode < 200 || resp.StatusCode > 299 {
-		log.Printf("Warning: HTTP status code for request is %s\n", strconv.Itoa(resp.StatusCode))
+		log.Printf("Warning: HTTP status code for request is %s\n",
+			strconv.Itoa(resp.StatusCode))
+		if resp.StatusCode == 401 {
+			return errors.New("Unable to authenticate with Quobyte API service")
+		}
+		body, err := ioutil.ReadAll(resp.Body)
+		if err != nil {
+			return (err)
+		}
+		log.Printf("Warning: Dumping full reply body:\n%s\n", string(body))
```

Comment:
@tallclair wdyt, this basically undoes the logging change?

Reply:
oh yeah, can we not do this? Techincally the new dial restrictions should mitigate it, but I'd rather keep the full response out of the logs.

Reply:
https://github.com/quobyte/api/pull/20